### DISCUSSION TITLE
AWS services

### DISCUSSION THREAD

**Discussion by shashwattpandeyy** [directus/directus#10973] [Archive]
Hi, i have to use AWS services for getting the signed URL but currently  `const storage = require("directus/storage").default;` is supporting this for AWS services earlier this is ` const storage = require("directus/dist/storage").default;` but what if in future if i update directus, so there might be a chance that the path is changed again so by this my code will break and again I have to set different path. So is there any way to get all the AWS services in directus?

**Comment by rijkvanzanten** [comment]
Couldn't you just `const aws = require('aws-sdk');`?
